avocado/utils/kernel.py

```diff
@@ -22,7 +22,8 @@
 import os
 import shutil
 import tempfile
-from distutils.version import LooseVersion  # pylint: disable=E0611
+
+from packaging import version as versobjc
```

Comment:
I'm not familiar with `packaging` but looks like it is a third-party library. I just use third-party libraries in the last case scenarios. Maybe we should investigate if setuptools has no modules for help here.

Reply:
I did a field survey and came to understand that `packaging` is a reliable alternative for versioning[0]. It has been maintained for a very long time by the Python Packaging Authority[1] themselves and has an exhaustive set of documentation[2]. It does need to be installed though so adding it to the requirements should allow passing the checks I think.

[0] https://packaging.pypa.io/en/latest/version.html
[1] https://pypi.org/project/packaging/
[2] https://packaging.pypa.io/en/latest/

Reply:
Avocado runs on multiple architectures and distros, I'm not sure if adding this dependency just for checking the kernel version on a specific library is the best approach.

During a quick search here, I found that packaging has been vendored into setuptools [1]. And we already have setuptools as a dependency. 

So maybe, we could use something like this:

```python3
from pkg_resources import packaging
packaging.version.parse(os.uname()[2]) > packaging.version.parse('2.2.2') 
```

[1] - https://github.com/pypa/setuptools/tree/a94ccbf404a79d56f9b171024dee361de9a948da/pkg_resources/_vendor/packaging